src/RiskModel.java

```diff
@@ -127,6 +127,23 @@ public void attack(Country attackingCountry, Country defendingCountry, int numOf
         return attackerDice;
     }
 
+    public ArrayList<Country> getPlayersAdjacentCountries(Country country){
+        ArrayList<Country> adjacentCountries = new ArrayList();
+        countryRecurse(adjacentCountries, country);
+        return adjacentCountries;
+
+    }
+
+    private void countryRecurse(ArrayList playerCountries, Country country){
+        playerCountries.add(country);
+        for (Country adjacent: country.getAdjacentCountries()){
+            //if owned by same player and not in list already add it
+            if(adjacent.getPlayer() == ((Country) playerCountries.get(0)).getPlayer() && !playerCountries.contains(adjacent)) {
```

Comment:
This should probably just be the country parameter instead of hardcoding .get(0)